### DISCUSSION TITLE
Flex 4.x support?

### DISCUSSION THREAD

**Discussion by YordanYanakiev** [ruffle-rs/ruffle#3937] [General]
Hello.
Most of the companies is stuck with Flex 4 projects, and some of them - honestly cant be easily reproduced in HTML5 or anything else. Actually they were, but the sacrifices was ginormous.

Will we see Flex 4.x support, and when we could expect if it is coming ?

**Comment by flashliberty5467** [comment]
dont expect it to come for a while i would recommend kasm browser or cheerpx for flash while your waiting for ruffle

https://www.kasmweb.com/index.html

https://leaningtech.com/cheerpx-for-flash/
